logic/src/commonMain/kotlin/com/wire/kalium/logic/data/register/RegisterAccountRepository.kt

```diff
@@ -0,0 +1,51 @@
+package com.wire.kalium.logic.data.register
+
+import com.wire.kalium.logic.NetworkFailure
+import com.wire.kalium.logic.functional.Either
+import com.wire.kalium.logic.wrapApiRequest
+import com.wire.kalium.network.api.user.register.RegisterApi
+import com.wire.kalium.network.api.user.register.RegisterResponse
+
+interface RegisterAccountRepository {
+    suspend fun requestEmailActivationCode(email: String, baseApiHost: String): Either<NetworkFailure, Unit>
+    suspend fun verifyActivationCode(email: String, code: String, baseApiHost: String): Either<NetworkFailure, Unit>
```

Comment:
What do you think of a more general name like `verfiyAccount`? In my opinion that would make it more readable.
`verifyAccoutn(email, verificationCode....`

Reply:
`verifyAccount` implies that there is an account which it's not the case